### PR TITLE
sticky banner added

### PR DESCRIPTION
**Pull request by wasifkareem**
Feat:  Add Sticky banner- Closes[#2383](https://github.com/keploy/keploy/issues/2383)

- Responsive for all screens
- Follows keploy design guidelines
- CTA buttons that opens extension page in new tab

 https://github.com/user-attachments/assets/5995f6b8-fb59-41e0-b263-bcb59d5941c2

@TvisharajiK  Ready for review

### CODE REVIEW

**Comment by wasifkareem** [comment]
Hi @TvisharajiK @nehagup  this is ready for review.

File: app/layout.tsx [hunk 1]
```diff
@@ -18,19 +19,25 @@ export interface Metadata {
...
 
 export const defaultMetadata: Metadata = {
-  title: 'Keploy | Open Source Stubs and API Test Generator for Developer',
-  keywords: "Integration testing, e2e Testing, ai testing, Unit Testing, API Testing, open source ai testing tool, Service Mocking, Dependency Mocking",
-  description: 'Keploy is AI based test case and stubs/mocks generator for integration and unit testing. 90% test coverage in minutes with open source testing tool',
+  title: "Keploy | Open Source Stubs and API Test Generator for Developer",
+  keywords:
+    "Integration testing, e2e Testing, ai testing, Unit Testing, API Testing, open source ai testing tool, Service Mocking, Dependency Mocking",
+  description:
+    "Keploy is AI based test case and stubs/mocks generator for integration and unit testing. 90% test coverage in minutes with open source testing tool",
```
Comment (shivamsouravjha) [inline app/layout.tsx:26]: why is this changed?

**Comment by TvisharajiK** [comment]
@wasifkareem Couple of suggestions : 

1. Try to look into a better design format
2. Make the banner above the navbar and do not take up so much space, adding icons looks good but try to not take up so much of the space 
3. Change the color of the entire tag so that it gains attention, right now it merges a lot with the background

File: app/layout.tsx (hunk 1 above)
Comment (wasifkareem) [reply app/layout.tsx:26]: Hi @shivamsouravjha , nothing changed here, thats just auto formatting

**Comment by wasifkareem** [comment]
Screenshot from 2024-10-30 21-41-09
Screenshot from 2024-10-30 21-42-16




Great Suggestions @TvisharajiK , new designs look much better now, PR ready for review. also happy diwali 🪔

**Comment by Hermione2408** [comment]
Hey @wasifkareem , your PR has conflicts can you please resolve them, also please sign off DCO

**Comment by wasifkareem** [comment]
@Hermione2408 , sorry, I have no time for  this, you are free to use my branch and push it, I had actually created pretty good design for keploy

**Comment by Hermione2408** [comment]
Thanks for the PR! This issue was resolved by another PR, so closing this. Appreciate your work on this!
